R/IDateTime.R

```diff
@@ -302,24 +302,27 @@ as.POSIXlt.ITime = function(x, ...) {
 #   lubridate routines do not return integer values.
 ###################################################################
 
+do_fastroutine = function(x)
```

Comment:
please another name, `do_fastroutine` can be anything. `tz_insensitive`, or `utc_compatible`, or anything that refer to function of it

Reply:
Waiting for #3781 (hence WIP label)... will probably drop this helper after that's merged (since we can definitely `as.integer` on `ITime` but not `POSIXct` case)